FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -681,9 +681,18 @@ extension ComposeViewController: UIImagePickerControllerDelegate, UINavigationCo
             showAlert(message: "files_picking_photos_error_message".localized)
             return
         }
-        contextToSend.attachments.append(attachment)
+        appendAttachmentIfAllowed(attachment)
         node.reloadSections(IndexSet(integer: 2), with: .automatic)
     }
+
+    private func appendAttachmentIfAllowed(_ attachment: ComposeMessageAttachment) {
+        let totalSize = contextToSend.attachments.reduce(0, { $0 + $1.size })
+        if totalSize > GeneralConstants.Global.attachmentSizeLimit {
```

Comment:
Shouldn't this be `totalSize + attachment.size > GeneralConstants.Global.attachmentSizeLimit` ?